Swift/Tests/CBLTestCase.swift

```diff
@@ -40,6 +40,8 @@ class CBLTestCase: XCTestCase {
     
     let otherDatabaseName = "otherdb"
     
+    var defaultCol: Collection?
```

Comment:
Sorry for a bit picky here. Can we use full name, `defaultCollection`? In CBL-C test, it's also `defaultCollection`.

Reply:
will do